ubyssey/static/src/styles/modules/widgets/_upcoming_events_horizontal.scss

```diff
@@ -0,0 +1,51 @@
+.c-widget-upcoming-events-horizontal {
+  background-color: #f5f5f5;
+  width: 100%;
+  margin: 15px 0;
+
+  box-sizing: border-box;
+  padding: 25px 50px;
+  border-top: solid 1px $color-border-gray;
+  border-bottom: solid 1px $color-border-gray;
+}
+
+.c-widget-upcoming-events-horizontal__label-row {
```

Comment:
Try this instead of floats:

```scss
.c-widget-upcoming-events-horizontal__label-row {
  display: flex;
}

.c-widget-upcoming-events-horizontal__label {
  flex: 1;
}

.c-widget-upcoming-events-horizontal__view-all {
  // No styles
}